apis/database/v1beta1/dbsubnetgroup_types.go

```diff
@@ -151,6 +159,21 @@ type DBSubnetGroupList struct {
 	Items           []DBSubnetGroup `json:"items"`
 }
 
+// DBSubnetGroupObservation is the representation of the current state that is observed
+type DBSubnetGroupObservation struct {
+	// SubnetGroupStatus specifies the current state of this DB subnet group.
+	SubnetGroupStatus string `json:"subnetGroupStatus,omitempty"`
+
+	// DBSubnetGroupArn is the Amazon Resource Name (ARN) for this DB subnet group.
+	DBSubnetGroupArn string `json:"dbSubnetGroupArn,omitempty"`
```

Comment:
I think `state` and `arn` would be sufficient for names as it's already implied with these fields being under DBSubnetGroup CRD. I see that they are indeed named as `DBSubnetGroupArn` and `SubnetGroupStatus` in cloud API but IMO we should go with `state` and `arn` as it's really unnecessary.